### DISCUSSION TITLE
真的没有人喜欢绝区零了吗

### DISCUSSION THREAD

**Discussion by Modog1145** [share121/inter-knot#1113] [其他]
![1722490985405](https://github.com/user-attachments/assets/72182840-984f-4a62-b891-d514d26a262b)

昨天，我在绳网发布了“绝区零我们喜欢你”这条帖子，但没有什么人观看
要知道，《绝区零》是一款由米哈游开发的ARPG游戏，游戏以卡带未来主义和90年代怀旧风格为背景，打造了一个独特而吸引人的世界。玩家在游戏中扮演的角色是兄妹组成的传奇绳匠“法厄同”，可以选择操控哥哥“哲”或妹妹“铃”。除此之外，玩家还可以操控已建立合作关系的“代理人”，通过调频与代理人建立合作关系，并且可以进行角色培养和信赖提升。
现在，我就将“绝区零我们喜欢你”这首歌再次送给大家，来咯
绝区零 绝区零 我们喜欢绝区零
真是好玩很好玩 真的很好玩
我们喜欢妮可 安比 比利 猫又 苍角
我们喜欢艾莲乔 可琳 莱卡恩
绝区零 绝区零 我们喜欢绝区零
真是玩得好开心 玩的好高兴
真是喜欢好喜欢 真的好喜欢
真是好棒的游戏 好玩的游戏
想让我唱什么歌，评论区告诉我哦

**Comment by Erdbronie** [comment]
我们绝区零也有自己的史赤辣🤓👆

**Comment by WORLD-executeME** [comment]
能为我最喜欢的录像店Random Play唱一首歌吗？

**Comment by ijog36** [comment]
水帖是吧§（*￣▽￣*）§

**Comment by lingyushen** [comment]
+3

**Comment by GanlinSama** [comment]
XX少年团，我们喜欢你~~~
